### DISCUSSION TITLE
how to make it generate manifest version 2 ?

### DISCUSSION THREAD

**Discussion by gedobox** [tmkx/webx-kit#109] [Q&A]
I am limited by my JS knowledge. 

changing manifest.ts gives me errors about "content_security_policy" because v2 is expected to have a string not an object.

i can do it by changing node modules, but there has to be the Right way.

**Comment by tmkx** [comment]
It's possible to support Manifest V2, but it's not the primary focus of this library. Chrome is deprecating MV2, what is the main reason for using it?

**Reply by npearson72** [reply]
I too would be interested in MV2 for Firefox support. Looks like a promising project.

**Reply by tmkx** [reply]
Thanks for your acknowledgment. However, as far as I know, Firefox is also in the process of migrating to MV3? I'm uncertain about the ROI of supporting the sunset MV2.

https://extensionworkshop.com/documentation/develop/manifest-v3-migration-guide/
